src/bentoml/_internal/utils/__init__.py

```diff
@@ -137,8 +137,14 @@ def first_not_none(*args: T | None, default: None | T = None) -> T | None:
     return next((arg for arg in args if arg is not None), default)
 
 
-def randomize_runner_name(module_name: str):
-    return f"{module_name.split('.')[-1]}_{uuid.uuid4().hex[:6].lower()}"
+def normalize_labels_value(label: dict[str, t.Any] | None) -> dict[str, str] | None:
+    if not label:
+        return label
+    if any(not isinstance(v, str) for v in label.values()):
+        logger.warning(
+            "'labels' should be a dict[str, str] to conform with Kubernetes label spec. Converting all values to string. See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for more information."
```

Comment:
It is a bit strange to require user to conform to Kubernetes label spec in BentoML context. We can simply say the requirement of string is enforced by BentoML, and not mention K8s.